conformance/reports/v0.5.1/gateway/envoy-ai-gateway/aigw-latest-report.yaml

```diff
@@ -0,0 +1,23 @@
+GatewayAPIInferenceExtensionVersion: v0.5.1
+apiVersion: gateway.networking.k8s.io/v1
+date: "2025-08-15T14:10:31-09:00"
+gatewayAPIChannel: experimental
+gatewayAPIVersion: v1.3.0
+implementation:
+  contact:
+  - github.com/envoyproxy/ai-gateway/issues/new/choose
+  organization: envoyproxy
+  project: envoy-ai-gateway
+  url: github.com/envoyproxy/ai-gateway
+  version: latest
```

Comment:
nit: is there a formal tag that can be used here instead of `latest`?
version `latest` is always kind of confusing.. e.g., if I look on this report in a year from now, I'd like to easily understand which envoy aigw is conformant with GIE v0.5.1.
maybe even pseudo version is fine.